manifests-maistra/1.0.2/maistraoperator.v1.0.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,421 @@
+      [
+        {
+          "apiVersion": "maistra.io/v1",
+          "kind": "ServiceMeshControlPlane",
+          "metadata": {
+            "name": "basic-install"
+          },
+          "spec": {
+            "istio": {
+              "global": {
+                "proxy": {
+                  "resources": {
+                    "requests": {
+                      "cpu": "100m",
+                      "memory": "128Mi"
+                    },
+                    "limits": {
+                      "cpu": "500m",
+                      "memory": "128Mi"
+                    }
+                  }
+                }
+              },
+              "gateways": {
+                "istio-egressgateway": {
+                  "autoscaleEnabled": false
+                },
+                "istio-ingressgateway": {
+                  "autoscaleEnabled": false
+                }
+              },
+              "mixer": {
+                "policy": {
+                  "autoscaleEnabled": false
+                },
+                "telemetry": {
+                  "autoscaleEnabled": false,
+                  "resources": {
+                    "requests": {
+                      "cpu": "100m",
+                      "memory": "1G"
+                    },
+                    "limits": {
+                      "cpu": "500m",
+                      "memory": "4G"
+                    }
+                  }
+                }
+              },
+              "pilot": {
+                "autoscaleEnabled": false,
+                "traceSampling": 100.0
+              },
+              "kiali": {
+                "enabled": true
+              },
+              "grafana": {
+                "enabled": true
+              },
+              "tracing": {
+                "enabled": true,
+                "jaeger": {
+                  "template": "all-in-one"
+                }
+              }
+            }
+          }
+        },
+        {
+          "apiVersion": "maistra.io/v1",
+          "kind": "ServiceMeshMemberRoll",
+          "metadata": {
+            "name": "default"
+          },
+          "spec": {
+            "members": [
+              "your-project",
+              "another-of-your-projects" 
+            ]
+          }
+        }
+      ]
+spec:
+  version: 1.0.2
+  maturity: alpha
+  displayName: Red Hat OpenShift Service Mesh
+  description: |-
+    Red Hat OpenShift Service Mesh is a platform that provides behavioral insight and operational control over the service mesh, providing a uniform way to connect, secure, and monitor microservice applications. 
+  keywords: [ 'istio', 'maistra', 'servicemesh' ]
+  maintainers:
+  - name: Red Hat, OpenShift Service Mesh
+    email: istio-feedback@redhat.com
+  provider:
+    name: Red Hat, Inc.
+  links:
+  - name: Service Mesh Operator
+    url: https://github.com/Maistra/istio-operator
+  - name: Istio
+    url: https://istio.io/
+  replaces: maistraoperator.v1.0.1
```

Comment:
Can it replace 1.0.0 as well?
Not sure how it works, this is just a general question...

Reply:
Rob asked that question on the other PR, from the docs it looks as if it doesn't need to since it forms a DAG from the versions/replaces information.  It looks as if it should be only a single value.

Reply:
I don't think so: https://github.com/operator-framework/operator-sdk/blob/master/doc/user/olm-catalog/generating-a-csv.md#csv-fields

Reply:
Better docs on this process

https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/how-to-update-operators.md